Libraries/src/Amazon.Lambda.RuntimeSupport/Helpers/Logging/DefaultLogMessageFormatter.cs

```diff
@@ -48,21 +50,53 @@ public override string FormatMessage(MessageState state)
                 message = ApplyMessageProperties(state.MessageTemplate, messageProperties, state.MessageArguments);
             }
 
+            var displayLevel = state.Level != null ? ConvertLogLevelToLabel(state.Level.Value) : null;
+
+            var sb = new StringBuilder(
```

Comment:
I'm trying to compute the initial capacity of the string builder to avoid extra allocations.